src/Raven.Server/Documents/Revisions/RevisionsStorage.cs

```diff
@@ -1496,22 +1496,28 @@ private async Task WriteRevertedRevisions(List<Document> list, OperationCancelTo
             list.Clear();
         }
 
-        private bool PrepareRevertedRevisions(DocumentsOperationContext writeCtx, Parameters parameters, RevertResult result, List<Document> list, OperationCancelToken token, string collection = null)
+        private bool PrepareRevertedRevisions(DocumentsOperationContext writeCtx, Parameters parameters, RevertResult result, List<Document> list, OperationCancelToken token, List<string> collections = null)
         {
             using (_database.DocumentsStorage.ContextPool.AllocateOperationContext(out DocumentsOperationContext readCtx))
             using (readCtx.OpenReadTransaction())
             {
                 IEnumerable<Table.TableValueHolder> tvrs = null;
```

Comment:
Initialize to `Enumerable.Empty<>`, so you don't need to do the check below

Reply:
Done